test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -3,13 +3,25 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
 
 import org.junit.Test;
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(0));
+        assertFalse(Utils.isAnyNull(new ArrayList<>()));
+        assertFalse(Utils.isAnyNull(new NullPointerException()));
+
+        assertTrue(Utils.isAnyNull(1,2,3,null));
+        assertTrue(Utils.isAnyNull(null, null));
+        assertTrue(Utils.isAnyNull((Object) null)); //test fails at this portion
+    }
+
```

Comment:
In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch